### DISCUSSION TITLE
Τοπική Αργία Πέμπτη 30/11/23

### DISCUSSION THREAD

**Discussion by xristostsilis** [upatras-hci/iv#150] [Q&A] [answered]
Την Πέμπτη 30/11 είναι η εορτή του Αγίου Ανδρέα, του πολιούχου της Πάτρας. Το μάθημα μας θα πραγματοποιηθεί για αυτήν την εβδομάδα; Αν όχι, πρέπει να ανεβάσουμε στο discussion και την εβδομάδα 8; Η θα την «οφείλουμε» να γίνει την επόμενη;

**Comment by IKaramitsos** [comment, marked as answer]
Καλησπέρα και από εμένα, σύμφωνα με το ακαδημαϊκό ημερολόγιο ισχύει η παρακάτω εικόνα : 
![image](https://github.com/upatras-hci/iv/assets/63364209/c6df9306-dccc-40d7-8f7f-61bb006a29ef)
Από εκεί και πέρα αναμένουμε απάντηση

**Reply by epidrome** [reply]
Αν και στην Κέρκυρα δεν είναι αργία υποθέτω ότι υπέρ-ισχύει το ημερολόγιο του οικοδεσπότη οπότε δεν θα γίνει μάθημα σήμερα!

Το εβδομαδιαίο πρόγραμμα του μαθήματος δεν επηρεάζεται από αυτήν την αργία οπότε για την επόμενη εβδομάδα έχουμε ένα ακόμη παραδοτέο.
